io/net/bonding.py.data/bonding.yaml

```diff
@@ -0,0 +1,9 @@
+Test: !mux
+    option:
+        bonding_mode: "0 1 2 3 4 5 6"
+Parameters:
+    Iface1: "enP3p1s0f0"
+    Iface2: "enP3p1s0f1"
+    peerip: "85.85.85.85"
+    peerif2: "eth3"
```

Comment:
Have Ifaces and peerifaces as list of interfaces in host and peer respectively, so that same script can be scaled for any no. of interfaces depending on multiplexer input.